### PR TITLE
Update the way of config`location-labels` if tiflash enable

### PR DESCRIPTION
**Pull request by nolouch**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@nolouch 请 involve 技术 review

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -25,6 +25,10 @@ aliases: ['/docs-cn/dev/tiflash/tiflash-configuration/','/docs-cn/dev/reference/
 
     - 使用 `pd-ctl -u <pd_ip:pd_port> store limit <store_id> <value>` 命令单独设置某个 store 的 Region 调度速度。（`store_id` 可通过 `pd-ctl -u <pd_ip:pd_port> store` 命令获得）如果没有单独设置，则继承 `store-balance-rate` 的设置。你也可以使用 `pd-ctl -u <pd_ip:pd_port> store limit` 命令查看当前设置值。
 
+- [`replication.location-labels`](/pd-control.md#config-show--set-option-value--placement-rules)：用来表示 TiKV 实例拓扑关系，其中 key 的序代表了不同标签的层次关系。在 tiflash 开启的情况下需要使用 `pd-ctl config placement-rules` 来设置默认值，详细可参考 [geo-distributed-deployment-topology](/geo-distributed-deployment-topology.md)。
+
+
+
```
Comment (TomShawn) [inline tiflash/tiflash-configuration.md:31]:
```suggestion
```

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -25,6 +25,10 @@ aliases: ['/docs-cn/dev/tiflash/tiflash-configuration/','/docs-cn/dev/reference/
 
     - 使用 `pd-ctl -u <pd_ip:pd_port> store limit <store_id> <value>` 命令单独设置某个 store 的 Region 调度速度。（`store_id` 可通过 `pd-ctl -u <pd_ip:pd_port> store` 命令获得）如果没有单独设置，则继承 `store-balance-rate` 的设置。你也可以使用 `pd-ctl -u <pd_ip:pd_port> store limit` 命令查看当前设置值。
 
+- [`replication.location-labels`](/pd-control.md#config-show--set-option-value--placement-rules)：用来表示 TiKV 实例拓扑关系，其中 key 的序代表了不同标签的层次关系。在 tiflash 开启的情况下需要使用 `pd-ctl config placement-rules` 来设置默认值，详细可参考 [geo-distributed-deployment-topology](/geo-distributed-deployment-topology.md)。
```
Comment (rleungx) [inline tiflash/tiflash-configuration.md:28]:
```suggestion
- [`replication.location-labels`](/pd-configuration-file.md#location-labels)：用来表示 TiKV 实例拓扑关系，其中 key 的序代表了不同标签的层次关系。在 tiflash 开启的情况下需要使用 [`pd-ctl config placement-rules`](/pd-control.md#config-show--set-option-value--placement-rules) 来设置默认值，详细可参考 [geo-distributed-deployment-topology](/geo-distributed-deployment-topology.md)。
```

File: tiflash/tiflash-configuration.md [hunk 3]
```diff
@@ -25,6 +25,8 @@ aliases: ['/docs-cn/dev/tiflash/tiflash-configuration/','/docs-cn/dev/reference/
 
     - 使用 `pd-ctl -u <pd_ip:pd_port> store limit <store_id> <value>` 命令单独设置某个 store 的 Region 调度速度。（`store_id` 可通过 `pd-ctl -u <pd_ip:pd_port> store` 命令获得）如果没有单独设置，则继承 `store-balance-rate` 的设置。你也可以使用 `pd-ctl -u <pd_ip:pd_port> store limit` 命令查看当前设置值。
 
+- [`replication.location-labels`](/pd-configuration-file.md#location-labels)：用来表示 TiKV 实例拓扑关系，其中 key 的序代表了不同标签的层次关系。在 tiflash 开启的情况下需要使用 [`pd-ctl config placement-rules`](/pd-control.md#config-show--set-option-value--placement-rules) 来设置默认值，详细可参考 [geo-distributed-deployment-topology](/geo-distributed-deployment-topology.md)。
```
Comment (TomShawn) [inline tiflash/tiflash-configuration.md:28]:
```suggestion
- [`replication.location-labels`](/pd-configuration-file.md#location-labels)：用来表示 TiKV 实例的拓扑关系，其中 key 的顺序代表了不同标签的层次关系。在 TiFlash 开启的情况下需要使用 [`pd-ctl config placement-rules`](/pd-control.md#config-show--set-option-value--placement-rules) 来设置默认值，详细可参考 [geo-distributed-deployment-topology](/geo-distributed-deployment-topology.md)。
```

**Review by rleungx — APPROVED** [review]

**Pull request merged**